src/components/pages/EditModal/EditModal.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import { editUser } from "../../../services/service";
+import "./edit-user.scss";
```

Comment:
```suggestion
import "./edit-modal.scss";
```
Or else we can rename `EditModal` -> `EditUser` component